Makefile

```diff
@@ -1,5 +1,7 @@
-.PHONY: start
+.PHONY: start consumer
 
 start:
-	docker-compose up
-	
\ No newline at end of file
+	docker-compose --profile api up
```

Comment:
I don't really understand the user of profile over using `docker-compose up api`

Reply:
Maybe it's a naming issue. The `api` profile is on both the front [gateway](https://github.com/alexdunne/gs-onboarding/pull/6/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R22) service and the [api](https://github.com/alexdunne/gs-onboarding/pull/6/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R47) service which interacts with the db. By adding a shared profile to these two services and a different one to the [consumer](https://github.com/alexdunne/gs-onboarding/pull/6/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R35) service we can run start the two "api" services separately from the consumer worker.

Completely unnecessary :D. I started looking at how to start them separately because the `consumer` worker is log happy. I could've just used `-d` to run the services in the background and used `docker logs` to filter on the logs I cared about.